### PR TITLE
tokio: Initial integration

### PR DESCRIPTION
**Pull request by silvergasp**
Signed-off-by: Nathaniel Brough <nathaniel.brough@gmail.com>

### CODE REVIEW

**Comment by silvergasp** [comment]
TODO: The git clone currently points to my fork of tokio. This should be switched back once https://github.com/tokio-rs/tokio/pull/5392 is merged.

**Review by jonathanmetzman — APPROVED** [review]
lgtm
We approve of adding this project.

**Comment by jonathanmetzman** [comment]
let me know when this is ready for review.

**Comment by silvergasp** [comment]
Will do! it's currently pointed at my fork, while I work through the kinks :)

**Comment by silvergasp** [comment]
CC @Darksonn

File: projects/tokio/project.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+homepage: "https://tokio.rs/"
+language: rust
+primary_contact: aliceryhl@google.com
```
Comment (Darksonn) [inline projects/tokio/project.yaml:3]: Just for the record: This is me.

**Review by Darksonn — APPROVED** [review]
LGTM.

**Pull request merged**